compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/BIRGen.java

```diff
@@ -1564,8 +1563,7 @@ public void visit(BLangTypeConversionExpr astTypeConversionExpr) {
 
     @Override
     public void visit(BLangStructLiteral astStructLiteralExpr) {
-        List<BIROperand> varDcls = mapToVarDcls(astStructLiteralExpr.enclMapSymbols);
```

Comment:
What was `enclMapSymbols` being used for?

Reply:
to keep closure variables(variable references used as default values).